### PR TITLE
Improve PT-BR strings and simplify update scripts

### PR DESCRIPTION
**Pull request by pedrozzz0**

### CODE REVIEW

**Comment by EXtremeExploit** [comment]
i think the portuguese translations are correct, will have to check for @Zangetsu38 for them just in case, altho i see no reason as to change the update scripts :thinking:, if possible could that be done in other PR so its easier to manage?

**Comment by Zangetsu38** [comment]
No understund this change on script

**Comment by pedrozzz0** [comment]
If you want i can close this and open another PR for the specific script commit tbh

File: lang/pt-br.xml [hunk 1]
```diff
@@ -255,7 +255,7 @@ Connect a controller that is compatible with SDL2.</not_connected>
 				<cancel_loading>Você quer cancelar o carregamento?</cancel_loading>
 				<no_saved_data>Não há dados salvos.</no_saved_data>
 				<load_complete>Carregamento completo.</load_complete>
-				<loading>Carregamento...</loading>
+				<loading>Carregando...</loading>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:258]: revert this

File: lang/pt-br.xml [hunk 2]
```diff
@@ -276,7 +276,7 @@ Não desligue o sistema ou feche o aplicativo.</warning_saving>
 
 	<game_data>
 		<app_close>O seguinte aplicativo será fechado.</app_close>
-		<data_delete>Dados a serem excludíos:</data_delete>
+		<data_delete>Dados a serem excluídos:</data_delete>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:279]: revert

Comment (pedrozzz0) [reply lang/pt-br.xml:279]: why though? those translations are right afaik

Comment (pedrozzz0) [reply lang/pt-br.xml:279]: "excludios" isn't even a word in portuguese

Comment (Zangetsu38) [reply lang/pt-br.xml:279]: @Croden1999 you can check please
@pedrozzz0 here some translate is basée on officiel vita string

Comment (Croden1999) [reply lang/pt-br.xml:279]: > @Croden1999 you can check please @pedrozzz0 here some translate is based on official vita string

i check iit again, no any problem.

File: lang/pt-br.xml [hunk 3]
```diff
@@ -215,10 +215,10 @@ depending on its size and your hardware.</app_delete_note>
 	</compatibility>
 
 	<content_manager name="Gerenciador de conteúdo">
-		<search>Search</search>
+		<search>Pesquisar</search>
 		<application name="Aplicativos">
 			<delete>Os aplicativos selecionados e todos os dados relacionados, inclusive dados salvos, serão excluídos.</delete>
-			<no_item>Não há nenhum itens de conteúdo.</no_item>
+			<no_item>Não há nenhum item de conteúdo.</no_item>
```
Comment (Croden1999) [inline lang/pt-br.xml:221]: revert it, it is based on psvita system language.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/pt-br.xml (hunk 3 above)
Comment (pedrozzz0) [reply lang/pt-br.xml:221]: what do i need to revert exactly?

Comment (Zangetsu38) [reply lang/pt-br.xml:221]: this line

Comment (pedrozzz0) [reply lang/pt-br.xml:221]: seems like he already did it, can we get this merged or just close this altogether? @Croden1999 @Zangetsu38

Comment (Croden1999) [reply lang/pt-br.xml:221]: > seems like he already did it, can we get this merged or just close this altogether? @Croden1999 @Zangetsu38

rebase and push

Comment (pedrozzz0) [reply lang/pt-br.xml:221]: done, i think so.

Comment (pedrozzz0) [reply lang/pt-br.xml:221]: I think i will just close this and open a new one with more cleaner tree anyways.

**Comment by Zangetsu38** [comment]
No close for that... juste squash and force push, no need create new pr

File: lang/pt-br.xml [hunk 4]
```diff
@@ -203,22 +206,22 @@ depending on its size and your hardware.</app_delete_note>
...
-			<state id = "1344751053">Menu</state>
-			<state id = "1344752299">No jogo -</state>
-			<state id = "1260231985">No jogo +</state>
-			<state id = "920344019">Jogável</state>
+			<state id = "-1">Desconhecido</state>
+			<state id = "0">Nada</state>
+			<state id = "1">Inicializável</state>
+		    <state id = "2">Intro</state>
+            <state id = "3">Menu</state>
+            <state id = "4">No jogo -</state>
```
Comment (Croden1999) [inline lang/pt-br.xml:214]: wrong newline

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/pt-br.xml [hunk 5]
```diff
@@ -430,10 +433,10 @@ Você pode conquistar troféus usando um aplicativo compatível com o recurso de
 Tem certeza de que deseja continuar?</user_delete_warn>
 		<user_deleted>Usuário excluído.</user_deleted>
 		<change_avatar>Alterar avatar</change_avatar>
-		<reset_avatar>Reset Avatar</reset_avatar>
+		<reset_avatar>Resetar avatar</reset_avatar>
 		<name>Nome</name>
-		<user>Usuario</user>
+		<user>Usuário</user>
```
Comment (Croden1999) [inline lang/pt-br.xml:438]: revert, is based ps4 lang

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Comment by HuanJiCanShang** [comment]
the pr is outdated

**Comment by Zangetsu38** [comment]
I close it, @Croden1999 if you can take all valide change on it and pr it

**Comment by Croden1999** [comment]
> I close it, @Croden1999 if you can take all valide change on it and pr it

got it, sir

**Pull request closed without merging**